#ifndef SIGNUP_H
#define SIGNUP_H

#include <QDialog>


namespace Ui {
class signup;
}

class signup : public QDialog
{
    Q_OBJECT

public:
    explicit signup(QWidget *parent = nullptr);
    ~signup();

private slots:
    void on_calendarWidget_activated(const QDate &date);

    void on_calendarWidget_clicked(const QDate &date);

    void on_label_6_linkActivated(const QString &link);

    void on_pushButton_2_clicked();

    void on_pushButton_clicked();

private:
    Ui::signup *ui;
};

#endif // SIGNUP_H
